config/routes.rb

```diff
@@ -36,4 +37,14 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinary_offices#veterinarians"
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
+
+  get "/applications/new", to: "adoption_application#new"
+  get "/applications/:id", to: "adoption_application#show"
+  post "/applications", to: "adoption_application#create"
+
+  get "/admin/shelters", to: "admin/shelters#index"
+
+  get "/admin/applications/:id", to: "admin/adoption_applications#show"
```

Comment:
Based on my comments from your controllers, there could be a couple other routes:

`post "/applications/:id/pets", to: "adoption_application_pets#create"` For adding a pet to an application (creating a joins record)
`patch "/applications/:app_id/pets/:id", to: "adoption_application_pets#update"` For approving or rejecting a pet on an app
`patch "/applications/:id", to: "adoption_applications#update"` for submitting an application and changing the status